app/models/article.rb

```diff
@@ -11,16 +11,25 @@
 #  created_at  :datetime         not null
 #  updated_at  :datetime         not null
 #  account_id  :integer          not null
-#  author_id   :integer
+#  author_id   :bigint
```

Comment:
Shouldn't we add locale to the article as well? If we need to search all the articles in the locale `en`, with this modeling we would have to find all the categories in the locale and find the articles in those categories.

This doesn't need to take from the user, it can be derived from the category during creation.

Thoughts?

Reply:
@pranavrajs 
We will have an extra column to save the data copied from the category, but we need to join with the categories with articles on the listing page if the user wants to apply search on it.

I am not in favor of redundant information columns until and unless the join and search are very heavy to handle. We can take this if we face any such issues.